multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -22,25 +22,31 @@ def parse_reports(self):
     for f in self.find_log_files('qualimap/bamqc/genome_results'):
         parse_genome_results(self, f)
     self.qualimap_bamqc_genome_results = self.ignore_samples(self.qualimap_bamqc_genome_results)
+    self.write_data_file(self.qualimap_bamqc_genome_results, 'multiqc_bamqc_genome_results')
 
     # Coverage - coverage_histogram.txt
     self.qualimap_bamqc_coverage_hist = dict()
     for f in self.find_log_files('qualimap/bamqc/coverage', filehandles=True):
         parse_coverage(self, f)
     self.qualimap_bamqc_coverage_hist = self.ignore_samples(self.qualimap_bamqc_coverage_hist)
+    self.write_data_file(self.qualimap_bamqc_coverage_hist, 'multiqc_bamqc_coverage_hist')
```

Comment:
Ah sorry, I wasn't clear. No need to save data from plots - this is done already from the plot code (if requested on the command line). The only thing this needs to be added for is metrics data. Especially any that doesn't make it in to the report.

Reply:
Sorry I'm a bit lost here - which parts are missing then?

Reply:
Nothing missing - you added too much! If you can _remove_ the lines where you export the plot data, that would be fab 👍 (anything with `_hist`)

Reply:
Done :-)